Source/JavaScriptCore/Configurations/libJavaScriptCore.xcconfig

```diff
@@ -41,3 +41,5 @@ STRIP_INSTALLED_PRODUCT = NO;
 
 EXCLUDED_SOURCE_FILE_NAMES = $(inherited);
 EXCLUDED_SOURCE_FILE_NAMES[sdk=iphone*] = $(inherited) framework.sb;
+
+OTHER_LIBTOOLFLAGS = -no_warning_for_no_symbols
```

Comment:
Do we really need this to be redeclared? Seems like this target should inherit from CommonBase.xcconfig.